processor/src/main/java/org/mapstruct/ap/internal/conversion/AbstractJavaTimeToStringConversion.java

```diff
@@ -74,24 +76,34 @@ private String parametersListForParsing(ConversionContext conversionContext) {
         // See http://docs.oracle.com/javase/tutorial/datetime/iso/format.html for how to format Dates
         StringBuilder parameterBuilder = new StringBuilder( "<SOURCE>" );
         if ( !Strings.isEmpty( conversionContext.getDateFormat() ) ) {
-            parameterBuilder.append( ", DateTimeFormatter.ofPattern( \"" )
-                            .append( conversionContext.getDateFormat() )
-                            .append( "\" )" );
+            parameterBuilder.append( ", " );
+            parameterBuilder.append( dateTimeFormatter( conversionContext ) );
```

Comment:
Isn't

```
.append( conversionContext.getDateFormat() )
.append( "\" )" );
```

missing here?

Reply:
Looks like it is missing, but the `dateTimeFormatter` method (already existing one) is adding this. Naming of the method is confusing as I always named the methods to get the reference name in this way.

Reply:
Yes you are right sorry. This should have used that method from before.